tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -714,12 +724,27 @@ Tensor subalpha(const Tensor& input_a, const Tensor& input_b, float alpha, const
 }
 
 // addalpha(input, other, alpha) = input + (alpha * other)
-Tensor _addalpha(const Tensor& input_a, const Tensor& input_b, float alpha, const MemoryConfig& output_mem_config) {
-    Tensor result = add(mul_unary(input_b, alpha, output_mem_config), input_a, std::nullopt, output_mem_config);
-    return result;
+Tensor _addalpha(
+    const Tensor& input_a,
+    const Tensor& input_b,
+    float alpha,
+    std::optional<Tensor> output_tensor,
+    const MemoryConfig& output_mem_config) {
+    if (output_tensor.has_value()) {
+        output_tensor = add(mul_unary(input_b, alpha, output_mem_config), input_a, std::nullopt, output_mem_config);
+        return output_tensor.value();
+    }
```

Comment:
You need to pass output_tensor as the optional output tensor argument to add for this to work

Reply:
Yes. now that #8394 is merged I can pass the output_tensor.